login.go

```diff
@@ -21,8 +21,7 @@ type LoginError struct {
 }
 
 type LoginResponse struct {
-	AuthenticationToken string `json:"authentication_token"`
```

Comment:
Why do you remove the `AuthenticationToken` here and in the `User` structure

Reply:
Because now User() is coming from the auth service. AuthenticationToken was a legacy field only provided by reigonal API, it does not exist anymore.

Reply:
The logic is that now to get the list of available zones, you need to be logged as all zones are not available to all users.

So to login it should not be based on a regional API, but on the global auth service